src/Calculator.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import NavBar from './NavBar';
+
+class Calculator extends Component {
```

Comment:
Why is this a class based Component? You are not using any of the methods or state within?